spec/system/claims/support/claims/view_a_claim_spec.rb

```diff
@@ -46,25 +39,6 @@ def and_i_click_on_claim(claim)
     click_on(claim.id)
   end
 
-  def i_can_see_the_details_of_a_draft_claim
```

Comment:
This spec is not needed anymore as the claim always will need to have a provider